.moai/specs/SPEC-KANBAN-RENAME-001/acceptance.md

```
@@ -0,0 +1,412 @@
+```
+→ two `OK` lines.
+
+**AC-KR-015** — Given the five sibling documents, When they are grepped for the old contract vocabulary, Then zero matches remain on either side.
+```bash
+grep -rn 'factory' \
+  .claude/skills/moai/workflows/moai.md \
+  .claude/skills/moai/workflows/run.md \
+  .claude/skills/moai/workflows/run/mode-orchestration.md \
+  .claude/skills/moai/workflows/sync/quality-gates-quality.md \
+  .claude/rules/moai/workflow/goal-directive.md
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)